packages/as-proto-gen/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "as-proto-gen",
-  "version": "0.2.4",
+  "version": "0.3.0-alpha.3",
```

Comment:
please don't change that - we use `auto` for releases and it will automatically bump the version :)